src/core/constants.js

```diff
@@ -22,8 +22,18 @@ export const VERSION =
  */
 export const P2D = 'p2d';
 /**
- * One of the two render modes in p5.js: P2D (default renderer) and WEBGL
- * Enables 3D render by introducing the third dimension: Z
+ * One of the two render modes in p5.js, most known for 3D rendering.
+ *
+ * Some differences and new features to expect compared to the default render mode <a href="/#/p5/P2D">P2D</a>:
+ * - The <strong>coordinate system</strong> is centered and uses an optional z-dimension
+ * - <strong>Textures</strong>, <strong>materials</strong>, and <strong>lighting</strong> can be applied to shapes
+ * - You can control the <strong>camera</strong> that views the scene
+ * - <strong>Text</strong> is drawn differently
+ * - You can use <strong>shaders</strong>
```

Comment:
I was thinking about adding links to the bold features, like `<a href="https://github.com/processing/p5.js/wiki/Getting-started-with-WebGL-in-p5#text"><strong>Text</strong></a> is drawn differently`

Reply:
Linking sounds good to me, for what it's worth!

Reply:
I agree that it is helpful to link out to more info for each of those topics and I appreciate this as a brief list of differences. I think that replacing the bold text with links though might not be very clear, I wonder if things can be reworded in a way that lets you describe where the information can be found. Something like:

`The coordinate system is centered and uses an optional z-dimension, which you can read more about in the <a href="">"Getting Started With WebGL"</a> tutorial in the Learn page.`

Additionally, it would help to include a little more text for "Text is drawn differently" and "You can use shaders". For text, the reference page for [text()](https://p5js.org/reference/#/p5/text) describes some of the issues, maybe you can rephrase it into something like:

`Text in WebGL requires opentype/truetype fonts loaded in your [preload()](https://p5js.org/reference/#/p5/preload) function using the [loadFont()](https://p5js.org/reference/#/p5/loadFont) method. More information about using text in this mode can be found in the [wiki](https://github.com/processing/p5.js/wiki/Getting-started-with-WebGL-in-p5#text).`

And it might be helpful to briefly define shader, as some users might not be familiar:

`WebGL mode makes it possible to use shaders, which are programs that can be used for creating a variety of effects and graphics that take advantage of hardware acceleration.`